test/e2e/data/infrastructure-docker/cluster-template/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+bases:
```

Comment:
nit. We can generate cluster-template.yaml on the fly and use that as a base to avoid extra changes we will need to do here when apis change.

Reply:
moved to #3651